### ISSUE TITLE
Wrong way to create a private tensor in `test_api_sockets.py`

### ISSUE THREAD

**Issue by bobsonlin26** [OpenMined/GridNode#15]
## Description
In `test_api_sockets.py`, private tensor is created as this way:
 `_x = x.private_tensor(allowed_users=("user"))`
However, `allowed_users` is declared as a `List` type in [the definition of private_tensor](https://github.com/OpenMined/PySyft/blob/7c6fed0f492e3252b988f1bf9c8c9b4519f79e6d/syft/frameworks/torch/tensors/interpreters/native.py#L790).

If we give `allowed_users` a tuple type object with a single string, for example , `allowed_users=("user")`, then at last its `allowed_users` will turn out to be:
![aa](https://user-images.githubusercontent.com/63451786/83333579-c9766480-a2d3-11ea-91e8-809a471bc23f.png)

We should give `allowed_users` a List type object instead of tuple type object

## How to Reproduce
```
import syft as sy
import torch as th
from syft.workers.node_client import NodeClient
import pdb

hook = sy.TorchHook(th)
alice = NodeClient(hook, "ws://<ip>:<port>", id="alice")
x = th.tensor([1.0, 0.4])

_x = x.private_tensor(allowed_users=("user"))
print(_x.child.allowed_users)
```

## Expected Behavior
Get a wrong `allowed_users`

## System Information
 - OS: Ubuntu 18.04
 - Language Version: Python 3.6
 - Package Manager Version: conda 4.7.12

**Comment by jmaunon** [comment]
Hi @bobsonlin26 , I am workin on this, It is just a minor update

**Comment by IonesioJunior** [comment]
Closed by #16
